extensions/amp-subscriptions/validator-amp-subscriptions.protoascii

```diff
@@ -68,4 +68,33 @@ tags: {  # amp-subscriptions-google
   requires_extension: "amp-subscriptions"
   attr_lists: "common-extension-attrs"
 }
+tags: {  # script
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "script tag that contains encrypted material"
+  mandatory_parent: "CONTENT"
```

Comment:
What's the "CONTENT" parent? Sorry, not super familiar with validator rules. Is it an actual tag name? We don't have "CONTENT" tags anywhere that I know of...

Reply:
I think that's meant to be a parent with a `subscriptions-section="content"` attribute but I'm not sure how that is represented in validator

Reply:
That's what I'd expect, but it doesn't look like validator rules currently reflect that.

Reply:
resolved offline

Reply:
resolved offline